Formula/termsql.rb

```diff
@@ -0,0 +1,22 @@
+class Termsql < Formula
+  desc "Convert text into SQL table and query it"
+  homepage "https://tobimensch.github.io/termsql/"
+
+  url "https://github.com/tobimensch/termsql/archive/22501e5c982ec04229643802e36756feb4687e89.tar.gz"
+  version "22501e5c982ec04229643802e36756feb4687e89"
+  sha256 "d8fabe96f57153a2e2852995d66f4506cf593bae4a6b9d7d76d3aef6150fb07e"
+
+  depends_on :python => ["sqlite3", "sqlparse"]
+
+  def install
+    # Replace path to man pages in hard-coded setup.py
+    inreplace "setup.py", %r{/usr/share/man/man1/}, man1
```

Comment:
@bfontaine I think I understood that, I guess I'm not clear on what the problem is. Looking at the [file itself](https://github.com/tobimensch/termsql/blob/master/setup.py):

```
data_files=[('/usr/share/man/man1/', ['termsql.1.gz'])]
```

should be changed to something like this:

```
data_files=[('/usr/local/Cellar/termsql/0.3/share/man/man1', ['termsql.1.gz'])]
```

There's no trailing slash there, but it shouldn't matter to `setup.py` according to the spec above. Please correct me if I'm misunderstanding something!